core/tracker/inclusion.go

```diff
@@ -228,6 +229,14 @@ func checkAttestationInclusion(sub submission, block block) (bool, error) {
 func reportMissed(ctx context.Context, sub submission) {
 	inclusionMisses.WithLabelValues(sub.Duty.Type.String()).Inc()
 
+	// TODO(corver): Remove debug logs for https://github.com/ObolNetwork/charon/issues/2130
+	b, _ := json.Marshal(sub) //nolint:errchkjson
+	log.Debug(ctx, "Debug missed submission details",
+		z.Any("pubkey", sub.Pubkey),
+		z.Any("duty", sub.Duty),
+		z.Hex("submission", b),
```

Comment:
the aim is to get the actual submission hex, then to write a script that downloads the actual block that contains the attesation and then to see why it didn't match.